specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2270 @@
+            "required": false,
+            "type": "string",
+            "pattern": "^\d+(-\d+)?$"
+          },	  
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Request is queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string",
+                "description": "URL containing the resultId used to track the progress and obtain the result of the analyze operation."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze receipt": {
+            "$ref": "./examples/InvoiceBatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/prebuilt/invoice/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Analyze Invoice Result",
+        "description": "Track the progress and obtain the result of the analyze invoice operation.",
+        "operationId": "GetAnalyzeInvoiceResult",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "description": "Analyze operation result identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeOperationResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get analyze receipt result": {
+            "$ref": "./examples/InvoiceBatchResult.json"
+          }
+        }
+      }
+    },
+    "/prebuilt/id/analyze": {
+      "post": {
+        "summary": "Analyze ID",
+        "description": "Extract field text and semantic values from a given ID document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png', 'image/tiff' or 'image/bmp'. Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
+        "operationId": "AnalyzeIdAsync",
+        "consumes": [
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
```

Comment:
Curious, so this means that locale and pageRange are not yet supported for this API? Will they be in the future so it is similar to other prebuilts?

Reply:
preview.3 supports pageRange. For Locale please sync up with Paul\Thomas offline.

Reply:
ID documents does not support locale hints in this release since we only support en-US driver licenses and all passports.  We may consider support for such hints in the future.